packages/elements-dev-portal/src/components/NodeContent/NodeContent.tsx

```diff
@@ -5,69 +5,52 @@ import {
   MockingProvider,
   ReferenceResolver,
 } from '@stoplight/elements-core';
+import { DocsProps } from '@stoplight/elements-core/components/Docs';
```

Comment:
should we export this typing from root of core?

Reply:
Yeah good idea, done in b5f425b339af79a62366c12070eb8e1a9c2aea26